modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -1168,4 +1179,140 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     return { signatureShareRound3 };
   }
   // #endregion
+
+  // #region self-managed cold utils
+
+  public async keyGenSMCRound1(
```

Comment:
```suggestion
  public async keyGenOVCRound1(
```

Since this is so specific to the OVC format, i'd prefer if we rename it to OVC so it can be searched for easily.